### DISCUSSION TITLE
Welcome to karakeep-app Discussions!

### DISCUSSION THREAD

**Discussion by MohamedBassem** [karakeep-app/karakeep#70] [Announcements]
## 👋 Welcome!

We're going to be using github discussions to share announcements, brainstorm ideas, and answer user questions. If you prefer a more realtime discussion method, you can come and chat with us on our small discord server [here](https://discord.gg/NrgeYywsFh).

**Comment by NLZ** [comment]
I guess this needs to be renamed as well
